psi4/driver/procrouting/response/scf_response.py

```diff
@@ -681,4 +685,101 @@ def tda_solver(e, n, g, m):
 
     core.print_out("\n")
 
+# Print out TDM vectors just length representation for now: other forms could be added)
+    core.print_out("Electric Transition Dipole Moments (Length) (AU):\nState       X          Y          Z\n")
+    for i, x in enumerate(_results):
+        core.print_out(f" {i+1: 4} {x.edtm_length[0]:< 10.6f} {x.edtm_length[1]:< 10.6f} {x.edtm_length[2]:< 10.6f}\n")
```

Comment:
Could you unpack x.etdm_length to make this more readable.

e.g.
```
for i, q in enumerate(_results):
    x, y, z = q.edtm_length
    core.print_out(f" {i+1: 4d} {x:< 10.6f} {y:< 10.6f} {z:< 10.6f}\n")
```

Reply:
Done.